src/pages/Commits.tsx

```diff
@@ -13,14 +15,40 @@ import {
 import { GET_MAINLINE_COMMITS } from "gql/queries";
 import { usePageTitle, useNetworkStatus } from "hooks";
 import { PageDoesNotExist } from "pages/404";
-import { ProjectFilterOptions } from "types/commits";
-import { CommitsWrapper } from "./commits/CommitsWrapper";
+import { CommitsWrapper } from "pages/commits/CommitsWrapper";
+import {
+  ChartToggleQueryParams,
+  ChartTypes,
+  ProjectFilterOptions,
+} from "types/commits";
 import { ProjectSelect } from "./commits/projectSelect";
 
+const DEFAULT_CHART_TYPE = ChartTypes.Absolute;
+
 export const Commits = () => {
   const { projectId } = useParams<{ projectId: string }>();
   const options = { projectID: projectId };
   const dispatchToast = useToastContext();
+  const { search } = useLocation();
+  const [currentChartType, setcurrentChartType] = useState<ChartTypes>(
```

Comment:
[nit] adjust casing to `setCurrentChartType`

Reply:
nice catch